Source/WebCore/css/makeprop.pl

```diff
@@ -540,10 +540,8 @@ sub sortByDescendingPriorityAndName
 
     static AtomString* propertyStrings = new AtomString[numCSSProperties]; // Intentionally never destroyed.
     AtomString& propertyString = propertyStrings[index];
-    if (propertyString.isNull()) {
-        const char* propertyName = propertyNameStrings[index];
-        propertyString = AtomString(propertyName, strlen(propertyName), AtomString::ConstructFromLiteral);
-    }
+    if (propertyString.isNull())
+        propertyString = ASCIILiteral::fromLiteralUnsafe(propertyNameStrings[index]);
```

Comment:
A little surprised we can’t get `gperf` to generate the `""_s` syntax.

Reply:
I'll double check, maybe I can. I haven't tried.

Reply:
Yes, I was able to do it, much nicer, thanks.